app/layout.tsx

```diff
@@ -1,9 +1,10 @@
+import { Box } from "@chakra-ui/react";
 import type { Metadata } from "next";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
 import { lato, roboto_slab } from "./fonts";
+import "./global.css";
```

Comment:
I think it is always good to have a final stop of a globals.css. I was trying to customize the arrows the other day using the actual classes of the react slick components and it wasn't working, but it was before I discovered the import was gone. So, I vote for leaving it as an option just in case.